docs/examples.md

```diff
@@ -74,6 +74,14 @@ Password:
 $y$j9T$A0Y3wwVOKP69S.1K/zYGN.$S596l11UGH3XjN...
 ```
 
+If you don't have `mkpasswd`, or your copy doesn't support `--method`, you can run `mkpasswd` in a container:
+
+```
+$ podman run -ti --rm quay.io/coreos/mkpasswd --method=yescrypt
```

Comment:
I'd be in favor of directly replacing the straight `mkpasswd` example above and only keeping this new mkpasswd-in-podman snippet here.

That said, I couldn't see where this container is coming from. It would be useful to have a breadcrumb to its source repo, perhaps directly in this doc.

Reply:
That's more than enough, thanks!